Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -55,6 +55,8 @@ extension SettingsDictionary {
     }
 
     /// Sets `"CODE_SIGN_STYLE"` to `"Automatic"` and `"DEVELOPMENT_TEAM"` to `devTeam`
+    /// This should be your developer team id.
+    /// Locating your team id: https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/
```

Comment:
What do you think of adding docs for the parameter directly? Something like:

```suggestion
    /// - Parameters:
    ///   - devTeam: Your Apple Developer Team ID. See [here](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/) how you can find it.
```

Reply:
I think that's a great idea!